controllers/index.js

```diff
@@ -21,6 +21,8 @@ var router = require('express').Router();
 router.use('/amp-access', require('./amp-access'));
 router.use('/amp-user-notification', require('./amp-user-notification'));
 router.use('/amp-analytics', require('./amp-analytics'));
+router.use('/*/collect', require('./amp-analytics/proxy'));
```

Comment:
I'm intercepting the same path pattern used by google-analytics, so the only thing I have to do later is change the host and cid